StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -12,18 +12,44 @@ extension Client {
     /// - Parameter options: Query options (See `QueryOptions`)
     /// - Returns: A new instance of `ChannelController`.
     ///
-    public func channelController(for channelId: ChannelId, options: QueryOptions = .all) -> ChannelControllerGeneric<ExtraData> {
-        .init(channelQuery: .init(channelId: channelId, options: options), client: self)
+    public func channelController(for cid: ChannelId, options: QueryOptions = .all) -> ChannelControllerGeneric<ExtraData> {
+        .init(channelQuery: .init(cid: cid, options: options), client: self)
     }
     
-    /// Creates a new `ChannelController` for the channel with the provided id.
+    /// Creates a new `ChannelController` for the channel with the provided channel query.
     ///
     /// - Parameter channelQuery: The ChannelQuery this controller represents
     /// - Returns: A new instance of `ChannelController`.
     ///
     public func channelController(for channelQuery: ChannelQuery<ExtraData>) -> ChannelControllerGeneric<ExtraData> {
         .init(channelQuery: channelQuery, client: self)
     }
+
+    /// Creates a new `ChannelController` that will create new channel.
+    ///
+    /// - Parameters:
+    ///   - id: The id for the new channel.
+    ///   - type: Type for the new channel.
+    ///   - team: Team for new channel.
+    ///   - members: IDs for the new channel members.
+    ///   - invites: IDs for the new channel invitees.
+    ///   - extraData: Extra data for the new channel.
+    /// - Returns: A new instance of `ChannelController`.
+    public func channelController(
+        createChannelWithId id: String,
+        type: ChannelType,
```

Comment:
I don't think we should separate channel `id` and `type` here. We are trying to use `cid` everywhere. It has type `ChannelId` that could confuse developers that exists something else `id`. 
```suggestion
        createChannelWithId cid: ChannelId,
```
@VojtaStavik what do you think?

Reply:
Hmmm, I was against this originally, but the more I think about it, the more I think this actually makes sense 👍

Reply:
Makes sense for me as well

Reply:
Changed in https://github.com/GetStream/stream-chat-swift/pull/390/commits/01718fb52c4ed934f67558f70297e0395768c253